packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -60,6 +63,12 @@ export const FeaturesContextProvider = ({
         flags,
       ) as ShareVersion,
       authVersion: getFeatureValue(Features.AuthenticationVersion, flags),
+      squadVersion: getFeatureValue(
```

Comment:
What's the difference between `squadButton` and `squadVersion`?

Reply:
Oh I see now, it's the copy of the button. Do we experiment on that?

Reply:
It's mentioned on Slack as an idea so better be prepared for it even if we don't use it.